openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,61 @@
+"""
+Models to support persistent tasks.
+"""
+
+import json
+
+from django.db import models
+
+
+class FailedTask(models.Model):
```

Comment:
As discussed with @kevin, let's make this an abstract model so each feature can have it's own table of failed tasks.  Each feature will want to manage its failed tasks differently.